students/sbolsen/lesson04/mailroom2.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import time
+
+donors = {
+        'Jesse Johnson': [150, 345],
+        'Mary May': [1000, 750],
+        'Spencer Samuels': [50, 200, 1100],
+        'Zach Zillow': [85],
+        'Tina Thompson': [76, 250, 300]
+        }
+
+
+def donor_list():
+    for donor in donors:
+        print(donor)
+
+
+def prompt_user():
+    choice = input('Please select from the following: \n1) Send a Thank You \n2) Create a Report \n3) Send letters to everyone \n4) Quit\n')
```

Comment:
Watch the line lengths on these strings. You can do something like this for Pep8:
```
    choice = input('Please select from the following: \n1) Send a Thank You \n2) Create a Report \n3) Send letters to '
                   'everyone \n4) Quit\n')
```
Or use triple quoted strings to preserve line breaks in your strings